tasks/system_probe.py

```diff
@@ -605,6 +606,57 @@ def ninja_generate(
         ninja_cgo_type_files(nw)
 
 
+def build_libpcap_static(ctx, dev_dir: str):
```

Comment:
i think the name of this function is wrong or confusing, why does it return the cgo_flags?

Reply:
True this function was poorly named, I changed it with cf234039 to just build libpcap, and made it so that the `get_libpcap_cgo_flags` function is the one returning the cgo_flags